cmk/base/plugins/agent_based/checkpoint_connections.py

```diff
@@ -7,18 +7,71 @@
 # .1.3.6.1.2.1.1.1.0 Linux gateway1 2.6.18-92cp #1 SMP Tue Dec 4 21:44:22 IST 2012 i686
 # .1.3.6.1.4.1.2620.1.1.25.3.0 19190
 
-from .agent_based_api.v1 import register, SNMPTree
-from .utils import checkpoint
+from typing import NamedTuple
 
+from cmk.base.plugins.agent_based.agent_based_api.v1 import (
+    check_levels,
+    register,
+    Service,
+    SNMPTree,
+)
+from cmk.base.plugins.agent_based.agent_based_api.v1.type_defs import CheckResult, DiscoveryResult
+from cmk.base.plugins.agent_based.utils import checkpoint
+
+checkpoint_connections_default_levels = {"levels": (40000, 50000)}
```

Comment:
This is not useful here. Instead, add 
`check_default_parameters= {"levels": (40000, 50000)}`
to `register.check_plugin`.
Furthermore, in order to be able to still resolve older autocheck files, please add
`checkpoint_connections_default_levels = (40000, 50000)`
to
cmk/utils/migrated_check_variables.py.